### PR TITLE
Added command and arg parsing

### PR DESCRIPTION
**Pull request by jjcollinge**
Addresses #147

### CODE REVIEW

**Comment by lawrencegripper** [comment]
@jjcollinge good to go on this one or you got more to add?

**Comment by jjcollinge** [comment]
Good to go for initial implementation. Using just Flagset makes it not perfect but if we add anything more advanced we can move to cobra or something

File: go.sum [hunk 1]
```diff
@@ -74,6 +74,8 @@ github.com/google/go-github v17.0.0+incompatible h1:N0LgJ1j65A7kfXrZnUDaYCs/Sf4r
 github.com/google/go-github v17.0.0+incompatible/go.mod h1:zLgOLi98H3fifZn+44m+umXrS52loVEgC2AApnigrVQ=
 github.com/google/go-querystring v1.0.0 h1:Xkwi/a1rcvNg1PPYe5vI8GbeBY/jrVuDX5ASuANWTrk=
 github.com/google/go-querystring v1.0.0/go.mod h1:odCYkC5MyYFN7vkCjXpyrEuKhc/BUO6wN/zVPAxq5ck=
+github.com/google/subcommands v1.0.1 h1:/eqq+otEXm5vhfBrbREPCSVQbvofip6kIz+mX5TUH7k=
```
Comment (lawrencegripper) [inline go.sum]: Needs a `go mod tidy` needed :)

Comment (jjcollinge) [reply go.sum]: done

**Pull request merged**